### PR TITLE
improve mobile response for description.tsx and mentorship.tsx

### LINKED ISSUE #89: Improve Mobile Responsiveness for Description.tsx and Mentorship.tsx
image

Test component with "F12" on test page for mobile responsive.
MAKE SURE THE ORIGINAL SCREEN LOOKS THE SAME AS BEFORE!!!! YOU SHOULD ONLY CHANGE THE LOOK ON MOBILE!!!

Be creative, as long as it looks reasonably nice.

Make sure to put md: in front of the formatting for regular screen size before you add a new one for mobile.
For example, if you want to have a smaller text size in mobile, put "md:" in front of text-xl and add a new text size.
"text-sm md:text-xl"

Be careful!! There are some default tailwind formatting! For example, in the original screen, there is no margin, so the default is m-0. If you want to add a margin to the mobile screen, make sure to not only add a margin but also to put "md: m-0" or it will apply to both regular screen and mobile.
"m-2 md:m-0"

PATH:
src\components\about\Mentorship.tsx
src\components\about\Description.tsx

### PR DESCRIPTION
**Pull request by inegishi**

### CODE REVIEW

**Comment by inegishi** [comment]
image

File: src/components/about/Description.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ const Description = () => {
         alt="star"
         className="absolute bottom-[-50%] right-[20%] w-[3.5vw]"
       />
-      <p className="mt-[2%] w-[60%] font-hearts text-[1.8vw] font-light text-hearts-brown">
+      <p className="mt-[2%] w-[60%] font-hearts text-[3vw] font-light text-hearts-brown lg:text-[1.8vw]">
```
Comment (qhgill) [inline src/components/about/Description.tsx]: check experience and todos components in about section and try to match width/text size

**Review by qhgill — CHANGES_REQUESTED** [review]
when checking for mobile make sure the page still looks uniform, should match components above, if you need help let me know.

**Comment by inegishi** [comment]
Screenshot 2024-11-23 204833

**Review by qhgill — APPROVED** [review]

**Pull request merged**